utils/bin/lvim.ps1

```diff
@@ -0,0 +1,8 @@
+$env:HOME = "$HOME"
+$env:USER = "$HOME"
+$env:SHELL = "cmd"
```

Comment:
I'm not sure this one is a good idea since PowerShell exists.

Reply:
Powershell can't open with floatterm. Only cmd do